imports/api/parties/server/publications.js

```diff
@@ -2,7 +2,7 @@ import { Meteor } from 'meteor/meteor';
 import Parties from '../parties_collection.js';
 
 Meteor.publish('approvedParties', function () {
-  return Parties.find({approved: true});
+  return Parties.find({ approved: true, startTime: { $gt: new Date() } });
```

Comment:
Commits like this are too small. Please combine small commits like this in the future. You can do that through creating a new branch and doing an interactive rebase.